react_main/src/css/vStyle.css

```diff
@@ -18,3 +18,30 @@
 .game .misc-wrapper .setup {
   max-width: 400px;
 }
+
```

Comment:
i think we should add all this css to setup.css instead of creating a separate vStyle.css file

it is easier to maintain